server/src/test/java/org/elasticsearch/search/aggregations/bucket/geogrid/GeoGridTestBase.java

```diff
@@ -0,0 +1,158 @@
+/*
+ * Licensed to Elasticsearch under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.elasticsearch.search.aggregations.bucket.geogrid;
+
+import org.apache.lucene.index.IndexWriter;
+import org.elasticsearch.search.aggregations.InternalAggregations;
+import org.elasticsearch.test.InternalMultiBucketAggregationTestCase;
+import org.elasticsearch.search.aggregations.ParsedMultiBucketAggregation;
+import org.elasticsearch.search.aggregations.pipeline.PipelineAggregator;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static org.hamcrest.Matchers.equalTo;
+
+public abstract class GeoGridTestBase<B extends InternalGeoGridBucket, T extends InternalGeoGrid<B>>
+    extends InternalMultiBucketAggregationTestCase<T> {
+
+    private int defaultPrecision;
```

Comment:
defaultPrecision seems to be only used once and is an arbitrary valid precision, do you think this can be replaced with an abstract `randomPrecision` as is done in the GeoGridAggregatorTest?

Reply:
yes, because the choice of this precision value is arbitrary. a specific one was used, but the `4` held no importance beyond being a random valid value

Reply:
done.